warehouse/static/images/LICENSE

```diff
@@ -0,0 +1,6 @@
+All images in this directory are copyrighted by the Python Software Foundation
+unless otherwise noted, and must be used in accordance with the PSF logo and
+trademark guidelines:
+
+    https://www.python.org/community/logos/
+    https://www.python.org/psf/trademarks/
```

Comment:
@ewdurbin Do you think this is sufficient?

Reply:
I believe so